consumer/file.go

```diff
@@ -140,351 +139,118 @@ func init() {
 
 // Configure initializes this consumer with values from a plugin config.
 func (cons *File) Configure(conf core.PluginConfigReader) {
-	cons.SetRollCallback(cons.onRoll)
-
-	var err error
-	cons.source, err = newSourceFile(conf)
-	conf.Errors.Push(err)
+	cons.done = make(chan struct{})
 
-	cons.seeker = newSeeker(conf)
+	// TODO: support manual roll again
+	//cons.SetRollCallback(cons.onRoll)
+	cons.SetStopCallback(func() {
+		close(cons.done)
+	})
 
 	// restore default observer mode for invalid config settings
 	if cons.observeMode != observeModePoll && cons.observeMode != observeModeWatch {
-		cons.Logger.WithField("observeMode", cons.observeMode).Errorf("Unknown observe mode '%s'", cons.observeMode)
+		cons.Logger.Warningf("Unknown observe mode '%s'. Using poll", cons.observeMode)
```

Comment:
If it's fine for you I'd like to not start the log message with a capital letter, as is idiomatic for error messages.

Reply:
I'd also like to do the same for other error/log messages contained in this pull request.

Reply:
Most log messages in Gollum start with an uppercase letter.
Is this really idiomatic? Gofmt complains on a `.` at the end of a message, but I don't know of any other convention (source?).

Reply:
https://github.com/golang/go/wiki/Errors#user-content-errors

Reply:
:+1: but again - as this has not been done throughout the code it should be a cleanup PR by itself (after #231 has been merged please).